webapp/src/main/java/io/github/microcks/web/MockControllerCommons.java

```diff
@@ -336,4 +337,20 @@ public static void publishMockInvocation(ApplicationContext applicationContext,
       applicationContext.publishEvent(event);
       log.debug("Mock invocation event has been published");
    }
+
+   public static String composeServiceAndVersion(String serviceName, String version) {
+      return "/" + UriUtils.encodeFragment(serviceName, "UTF-8") + "/" + version;
+   }
+
+   public static String extractResourcePath(HttpServletRequest request, String serviceAndVersion) {
+      String requestURI = request.getRequestURI();
+      String resourcePath = requestURI.substring(requestURI.indexOf(serviceAndVersion) + serviceAndVersion.length());
+      log.debug("Found resourcePath: {}", resourcePath);
+
+      // If resourcePath was encoded with '+' instead of '%20', replace them .
+      if (resourcePath.contains("+")) {
+         resourcePath = resourcePath.replace("+", "%20");
+      }
+      return resourcePath;
+   }
```

Comment:
Nice addition! It will be easy to reuse in other controller for rest and graphql too! 👍